app/infrastructure/broadcaster.py

```
@@ -0,0 +1,45 @@
+
+    async def publish(self, metric: dict) -> None:
+        async with self._lock:
+            if not self._subscribers:
+                return
+            subscribers = list(self._subscribers)
+
+        for queue in subscribers:
+            try:
+                queue.put_nowait(metric)
+            except asyncio.QueueFull:
+                pass
```

Ruff (0.12.2)

Use `contextlib.suppress(asyncio.QueueFull)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(asyncio.QueueFull)`

(SIM105)